Tweeter/Scripts/app.js

```diff
@@ -9,15 +9,26 @@
         console.log(response.exists);
         if (!response.exists) {
             $("#username-ans").addClass("glyphicon-ok");
+
+            $("#RegisterButton").attr("disabled", false);
+            console.log("username did not exist in database");
+            $('#RegisterButton').show();
+
         } else {
             $("#username-ans").addClass("glyphicon-remove");
+            
+            $("input[RegisterButton]").attr("disabled", true);
+            $("input[RegisterButton]").prop("disabled", true);
+            $('#RegisterButton').hide();
```

Comment:
You don't need to disable *and* hide it -- you probably could pick one or the other.